GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -179,49 +179,34 @@ function CreateVFSForGitDistribution()
 function CreateMetaDistribution()
 {
     GITVERSION="$($VFS_SCRIPTDIR/GetGitVersionNumber.sh)"
-    GITDMGPATH="$(find $VFS_PACKAGESDIR/gitformac.gvfs.installer/$GITVERSION -type f -name *.dmg)" || exit 1
-    GITDMGNAME="${GITDMGPATH##*/}"
-    GITINSTALLERNAME="${GITDMGNAME%.dmg}"
-    GITVERSIONSTRING=`echo $GITINSTALLERNAME | cut -d"-" -f2`
-    GITPKGNAME="$GITINSTALLERNAME.pkg"
-    
+    GITINSTALLERPKGPATH="$(find $VFS_PACKAGESDIR/gitformac.gvfs.installer/$GITVERSION -type f -name *.pkg)" || exit 1
```

Comment:
I've not been following much of the installer discussions, what are the pros of a .pkg (over a .dmg)?